.github/workflows/iroha2-dev-pr.yml

```diff
@@ -145,7 +146,15 @@ jobs:
       - name: Mark binaries as executable
         run: |
           chmod +x ${{ env.CLIENT_CLI_DIR }}
-      - name: Install dependencies using Poetry
+      - name: Install torii api dependencies using Poetry
+        working-directory: torii/pytests
+        run: |
+          poetry install
```

Comment:
Is it necessary to run `poetry install` inside the working directory? Because we already have a `poetry install` [command](https://github.com/hyperledger/iroha/blob/f12077d355b4c3fbfcd4da05638567142c8c2e56/Dockerfile.build#L19) in `Dockerfile.build`. But perhaps this one for `client-cli-tests`? If yes, we have to try to unify the `poetry install`  command. If it's not possible, we should get rid of this command in `Dockerfile.build`.

Reply:
I've tried to delete `poetry install` from here, but it's didn't work without it. 
![image](https://github.com/hyperledger/iroha/assets/111361420/96338b18-5940-4ae2-9b58-25942d292097)